version.go

```diff
@@ -1,7 +1,7 @@
 package beekeeper
 
 var (
-	version = "0.25.0" // manually set semantic version number
```

Comment:
this manual updating can be automated with something like this from Bee https://github.com/ethersphere/bee/blob/master/Makefile#L23